app/src/main/java/com/globant/harrypotterapp/contract/SpellsContract.kt

```diff
@@ -0,0 +1,12 @@
+package com.globant.harrypotterapp.contract
```

Comment:
move this package inside the viewmodel package pls

Reply:
Done!